canonical/ubuntu.com
# [WD-10513] update download raspberry pi

## Done

Removed word "certified" from "Raspberry Pi Zero 2 W" line for 24.04 LTS release on /download/raspberry-pi page according to the [copy doc](https://docs.google.com/document/d/1KgnSBa8mbR7qwlxxtqbE5a1wXh5GjtVums8TIB7IGVw/edit).
I will highlight the actual change with the comment, the rest is formatting by djLint.

## QA

- Navigate to https://ubuntu-com-13768.demos.haus/download/raspberry-pi
- Scroll down to "Supported devices" section
- Check that it says "Yes" next to "Raspberry Pi Zero 2 W" line for 24.04 LTS release

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10513

## Screenshots

![Screenshot 2024-04-18 at 16 26 41](https://github.com/canonical/ubuntu.com/assets/15943863/dce1ff55-6fe5-4129-8093-361ea7389fd7)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
